lib/cards.js

```diff
@@ -215,40 +213,43 @@ export const CARD_BRAND = {
    * @returns {Promise<Card[]>}
    * @tag Cards
    */
-   async list(accountID) {
-    checkString(accountID).or(Err.AccountID);
+  async list(accountID) {
+    checkString(accountID).or(Err.MISSING_ACCOUNT_ID);
 
     const result = await this.moov
       .got({
         url: `accounts/${accountID}/cards`,
-        method: "GET",
+        method: "POST",
```

Comment:
@sethlivingston The cards list endpoint is a GET, not a POST, right?
https://docs.moov.io/api/#operation/listCards

Reply:
Yes, thank you! I'm not sure how that happened.